mlir-tensorrt/python/bindings/Compiler/CompilerPyBind.cpp

```diff
@@ -19,7 +19,9 @@
 #include "mlir/Bindings/Python/PybindAdaptors.h"
 #include "pybind11/pybind11.h"
 #include "llvm/Support/DynamicLibrary.h"
+#include <iostream>
```

Comment:
This should be removed

Reply:
I will also need to remove the logging when we catch an exception in the Python callback in that case. Unless there is another logging mechanism in place?

Reply:
@christopherbate updated to use `llvm::errs` instead.